cli_flags.go

```diff
@@ -63,11 +63,10 @@ var (
 		"If zero, monotonic-realtime clock sync will be performed once, " +
 		"on agent startup, but not periodically."
 	sendErrorFramesHelp = "Send error frames (devfiler only, breaks Kibana)"
-	offCPUThresholdHelp = fmt.Sprintf("If set to a value between 1 and %d will enable "+
-		"off-cpu profiling: Every time an off-cpu entry point is hit, a random number between "+
-		"0 and %d is chosen. If the given threshold is greater than this random number, the "+
-		"off-cpu trace is collected and reported.",
-		support.OffCPUThresholdMax-1, support.OffCPUThresholdMax-1)
+	offCPUThresholdHelp = fmt.Sprintf("The per-mille chance for an off-cpu event being reported. "+
+		"Valid values are in the range [0..%d]."+
+		"Default is %d (off-cpu profiling turned off). ",
```

Comment:
Since we don't use make this configurable, also for consistency with line 97

```suggestion
		"Default is 0 (off-cpu profiling turned off). ",
```

Reply:
Not sure I can follow. The other default constants are also not configurable. They exist to have one source of truth and thus to make code maintenance easier.

Line 97 in config.go is unrelated, but could be rephrased to separate default value from "turning-off" value.